dipy/utils/defaults.py

```diff
@@ -0,0 +1,14 @@
+"""Module for default handing functions."""
```

Comment:
Happy to put the function wherever you prefer

Reply:
Just checking in on this, want to put this somewhere else? `mne` has a defaults.py in the root `mne` folder for file structure comparison if that helps at all and I think it is very well used and helps that it's a single place where all the default-type information is stored. It's not in `utils` though, which it might be better.

Reply:
How about putting this in `workflows.utils`?

Reply:
Done! No problem